src/mlpack/methods/softmax_regression/softmax_regression_impl.hpp

```diff
@@ -66,4 +68,4 @@ double SoftmaxRegression::Train(const arma::mat& data,
 } // namespace regression
 } // namespace mlpack
 
-#endif
+#endif
```

Comment:
Similar to src/mlpack/methods/softmax_regression/softmax_regression.cpp Line 145 comment.